src/clarity/forms/_forms.clarity.scss

```diff
@@ -17,6 +17,7 @@ $clr-form-disabled-background-color: $clr-light-midtone-gray !default;
 $clr-form-field-disabled-color: $clr-gray !default;
 $clr-checkbox-radio-height: $clr-icon-dimension-sm !default;
 $clr-form-icon-dim: $clr-icon-dimension-sm !default;
+$clr-checkbox-border-color: $clr-action-blue-light-midtone !default;
```

Comment:
nit: This variable name can be interpreted as the border color for all checkboxes. If this is just for the indeterminate one we should rename it to be a bit more specific.